apps/google-analytics-4/frontend/src/components/main-app/SlugWarningDisplay/SlugWarningDisplay.tsx

```diff
@@ -18,48 +17,40 @@ const SlugWarningDisplay = (props: Props) => {
   const contentTypeName = sdk.contentType.name;
 
   const openConfigPage = () => sdk.navigator.openAppConfig();
-  const linkToOpenConfigPage = (
-    <TextLink onClick={openConfigPage} target="_blank" rel="noopener noreferer">
-      {HYPER_LINK_MSG}
-    </TextLink>
+  const appConfigPageHyperLink = (bodyMsg: string) => (
+    <HyperLink onClick={openConfigPage} body={bodyMsg} substring="app configuration page." />
+  );
+
+  const supportHyperLink = (
+    <HyperLink
+      body={DEFAULT_ERR_MSG}
+      substring="contact support."
+      hyperLinkHref="https://www.contentful.com/support/?utm_source=webapp&utm_medium=help-menu&utm_campaign=in-app-help"
+    />
   );
 
   const { slugFieldIsConfigured, contentTypeHasSlugField, isPublished } =
     useSidebarSlug(slugFieldInfo);
 
-  const showHyperlink = !slugFieldIsConfigured || !contentTypeHasSlugField;
-  const { noSlugConfigMsg, noSlugContentMsg, notPublishedMsg } = getContentTypeSpecificMsg(
-    contentTypeName,
-    showHyperlink
-  );
+  const { noSlugConfigMsg, noSlugContentMsg, notPublishedMsg } =
+    getContentTypeSpecificMsg(contentTypeName);
 
   const renderContent = () => {
-    const content = { bodyMsg: DEFAULT_CONTENT_MSG, hyperLink: <></> };
+    const content: { bodyMsg: string | JSX.Element } = { bodyMsg: supportHyperLink };
     if (!slugFieldIsConfigured) {
-      content.bodyMsg = noSlugConfigMsg;
-      content.hyperLink = linkToOpenConfigPage;
+      content.bodyMsg = appConfigPageHyperLink(noSlugConfigMsg);
     } else if (!contentTypeHasSlugField) {
-      content.bodyMsg = noSlugContentMsg;
-      content.hyperLink = linkToOpenConfigPage;
+      content.bodyMsg = appConfigPageHyperLink(noSlugContentMsg);
```

Comment:
nit: I think you can combine these two if checks since they do the same thing like
```
   if (!slugFieldIsConfigured || !contentTypeHasSlugField) {
      content.bodyMsg = appConfigPageHyperLink(noSlugConfigMsg);
 }
```